pkg/apis/eventing/v1/broker_lifecycle_mt.go

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
I had actually been thinking about moving them away from our API directory alltogether and into the pkg/reconciler. Does not have to be part of this PR obviously :) But again, just sharing my thoughts.